nutanix/common_era_schema.go

```diff
@@ -0,0 +1,272 @@
+package nutanix
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	era "github.com/terraform-providers/terraform-provider-nutanix/client/era"
+)
+
+func timeMachineInfoSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeSet,
+		MaxItems:    1,
+		ForceNew:    true,
```

Comment:
This schema is not getting used anywhere. 
Also, we have marked it as ForceNew. Make sure that the caller funcs calling this func don't provision the parent resources after changing the machine details.
Please check the behavior of Force new or make it as a parameter to this function.